modules/app.js

```diff
@@ -0,0 +1,121 @@
+/* eslint-disable max-classes-per-file */
+/* eslint-disable no-use-before-define */
+const listOfbooks = document.querySelector('.books-content');
+const newB = document.querySelector('.new-books');
+const title = document.querySelector('#title');
+const author = document.querySelector('#author');
+
+class Book {
+  constructor(title, author) {
+    this.title = title;
+    this.author = author;
+    this.bookid = Math.random().toFixed(1);
+  }
+}
+
+class StoreBook {
+  constructor() {
+    // Array of objects for the book items
+    this.BookData = [];
+  }
+  // Add new Book to the book list
+
+  addBook(newbook) {
+    this.BookData.push(newbook);
+    localStorage.setItem('BookDB', JSON.stringify(this.BookData));
+    DisplayBooks(newbook);
+  }
+
+  // Remove book from the list
+  removeBook(bookid) {
+    const rmvbook = document.getElementById(bookid);
+    rmvbook.remove();
+    this.BookData = this.BookData.filter((x) => x.bookid !== bookid);
+    localStorage.setItem('BookDB', JSON.stringify(this.BookData));
+    removeSuccess();
+    if (this.BookData.length < 1) {
+      listOfbooks.classList.add('no-border');
+    }
+  }
+}
+
+const savebook = new StoreBook();
+// Get input value
+function getformInput() {
+  const insertbook = new Book(title.value, author.value);
+  return insertbook;
+}
+
+// Display teh list of books on the web page
+function DisplayBooks(index) {
+  let bgcolor = '';
+  if (savebook.BookData.indexOf(index) % 2 !== 0) {
+    bgcolor = 'white';
+  } else {
+    bgcolor = 'light';
+  }
```

Comment:
- I suggest using arrow functions since we are required to use ES6 syntax according to the project requirements, please fix this 🙏